frontend/packages/console-dynamic-plugin-sdk/src/extensions/actions.ts

```diff
@@ -92,6 +93,14 @@ export const isActionFilter = (e: Extension): e is ActionFilter => {
 
 // Support types
 
+interface CtaModal<P = any> {
+  type: 'modal';
```

Comment:
Nit - I think the `type` property can be removed since it doesn't appear to be used.